viz/log_plotter.py

```diff
@@ -1,5 +1,4 @@
 import matplotlib
```

Comment:
sorry I actually didn't know you were using this. I moved the LogPlotter to `training.py` because I thought it was the only place that used it. My bad, feel free to put it backin `utils`

Reply:
Although to be hnost I don't understand what `display_avg_KL` does ? Please explain (I mean we already store the test log and train log in `main.py`)